ambry-server/src/test/java/com/github/ambry/server/StatsManagerTest.java

```diff
@@ -136,7 +154,11 @@ public StatsManagerTest() throws Exception {
     ClusterMapConfig clusterMapConfig = new ClusterMapConfig(new VerifiableProperties(properties));
     storeMap = new HashMap<>();
     partitionToSnapshot = new HashMap<>();
-    preAggregatedSnapshot = generateRandomSnapshot().get(StatsReportType.ACCOUNT_REPORT);
+    List<Short> accountIdsToExlude = new ArrayList<>();
+    if (enableAccountExclusion) {
+      accountIdsToExlude.add((short) 0);
+    }
+    preAggregatedSnapshot = generateRandomSnapshot(accountIdsToExlude).get(StatsReportType.ACCOUNT_REPORT);
```

Comment:
Honestly, I feel like we probably don't need to preclude  account ids in `generateRandomSnapshot`. The "exclude accounts" logic is in BlobStoreStats and is already covered by `BlobStoreStatsTest`.  In the scope of StatsManager, I feel like verify the two lists of excluding accounts in constructor would suffice.

Reply:
OK, then I will remove the parameter and create a new test cases for account exclusion.